src/Stratis.Bitcoin.Features.BlockStore/BlockRepository.cs

```diff
@@ -509,6 +515,55 @@ public Task<Block> GetAsync(uint256 hash)
             return task;
         }
 
+        /// <summary>
+        /// Get multiple blocks from the database by their block hashes.
+        /// </summary>
+        /// <param name="hashes">The block hashes.</param>
+        public Task<List<Block>> GetBlocksAsync(List<uint256> hashes)
+        {
+            this.logger.LogTrace("({0}:'{1}')", nameof(hashes.Count), hashes.Count);
+            Guard.NotNull(hashes, nameof(hashes));
+
+            var results = new Dictionary<uint256, Block>();
+
+            Task<List<Block>> task = Task.Run(() =>
+            {
+                this.logger.LogTrace("()");
+
+                using (DBreeze.Transactions.Transaction transaction = this.DBreeze.GetTransaction())
+                {
+                    transaction.ValuesLazyLoadingIsOn = false;
+
+                    // Access hashes in sorted order.
+                    var byteListComparer = new ByteListComparer();
+                    hashes.Sort((pair1, pair2) => byteListComparer.Compare(pair1.ToBytes(), pair2.ToBytes()));
```

Comment:
I think we do not need special comparer, uint256 should be comparable

Reply:
The uint256's `ToBytes()` method performs a platform independent a big endian conversion which allows us to then use DBreeze's `ByteListComparer`. In that sense it does feel more intuitively correct although you are right in that using the default comparer will achieve the same.

Reply:
ok, makes sense to do it in a way DBreeze would do it